wiki/core/markdown/mdx/responsivetable.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import, unicode_literals
+
+import markdown
+from markdown.util import etree
+from markdown.treeprocessors import Treeprocessor
+
+
+class ResponsiveTableExtension(markdown.Extension):
+    """Wraps all tables with Bootstrap's table-responsive class"""
+
+    def extendMarkdown(self, md, md_globals):
+        md.treeprocessors.add('responsivetable', ResponsiveTableTree(md), "_end")
+
+
+class ResponsiveTableTree(Treeprocessor):
+    def run(self, root):
+        for table_wrapper in list(root.getiterator('table')):
+            table_new = self.create_table_element()
+            self.convert_to_wrapper(table_wrapper)
+            self.move_children(table_wrapper, table_new)
```

Comment:
Ah okay, so the existing `<table>` becomes a `<div>` and then a new `<table>` element is created and inserted?